framework/resource_router_set.go

```diff
@@ -0,0 +1,52 @@
+package framework
+
+import (
+	"github.com/giantswarm/microerror"
+)
+
+type ResourceRouterSetConfig struct {
+	ResourceRouters []*ResourceRouter
+}
+
+type ResourceRouterSet struct {
```

Comment:
Then (when ResourceRouter is called ResourceSet) this can be called simply ResourceRouter.

Reply:
Same, look comment above towards naming.